.github/workflows/check_migrations.yml

```diff
@@ -0,0 +1,53 @@
+name: Check migrations
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    container: python:3.12
+
+    services:
+      postgres:
+        image: postgres
+        env:
+          POSTGRES_USER: pguser
+          POSTGRES_PASSWORD: pgpass
+          POSTGRES_DB: consultations_test
+        options: >-
+          --health-cmd pg_isready
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 5
+        ports:
+          - 5432:5432
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Install python
+        uses: actions/setup-python@v2
+        with:
+          python-version: "3.12"
+
+      - name: Install poetry & deps
+        run: |
+          python -m pip install --upgrade poetry
+          poetry install --with development
+
+      - name: Check migrations
+        run: |
+          poetry run python manage.py migrate
+          poetry run python manage.py makemigrations --check
```

Comment:
Does this need to be its own action? (Could add to the existing `test` action as that shouldn't run in the first place if migrations are pending)

Reply:
Good point, can add to the test action.

Reply:
Now updated.